extension/json/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(JSONExtension)
+
+include_directories(yyjson/include)
+include_directories(include)
+add_subdirectory(yyjson)
+
+add_library(json_extension STATIC json-extension.cpp json_extract.cpp
+                                  json_type.cpp ${YYJSON_OBJECT_FILES})
+
+build_loadable_extension(json json-extension.cpp ${YYJSON_OBJECT_FILES})
```

Comment:
This should include all files related to the extension (i.e. including `json_extract.cpp` and `json_type.cpp`). I suspect this is causing the CI issues. I recommend moving the files into a variable similar to how this is done for the other extensions.

Reply:
Thanks! I've changed, this, still have to fix more CI stuff though before everything's green